docs/source/cn/random.py

```diff
@@ -0,0 +1,35 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.bernoulli,
+    r"""
+    bernoulli(x, *, generator=None, out=None)
+    
+    返回一个Tensor其参数为伯努利分布的二进制随机数(0 / 1) 。
```

Comment:
```suggestion
    返回一个 Tensor，起元素为0或1，且符合伯努利分布 。
```

这个需要了解下算子的用法，才可以知道翻译成 ”二进制“ 的不符合的。